static/manual_body.Rmd

```diff
@@ -1,237 +0,0 @@
----
```

Comment:
The reason for using .Rmd, aside from making it easier for others to edit (as stated by @jf1 ), is that we can show how to reproduce the results. Writing this html from scratch would take a long time!

![manual-eg](https://cloud.githubusercontent.com/assets/1825120/18998234/b08c7572-872f-11e6-8b19-bea0894661c0.png)